packages/terra-clinical-header/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Unreleased
 
 ### Changed
 * Update wdio spec to use Terra.describeViewports
+* Remove absolute positioning from title container
+* Add new theme variables for height
+** --terra-clinical-header-height
+** --terra-clinical-header-subheader-height
```

Comment:
These changes might make more sense under an "Added" section. Which ever section they end up in they might need a different syntax:

![image](https://user-images.githubusercontent.com/6720991/60832555-c9341e80-a181-11e9-9a95-c41129a266eb.png)

Reply:
updated https://github.com/cerner/terra-clinical/pull/529/commits/9bccacf8d5d239e892137d748dfbd6ec47a0039f